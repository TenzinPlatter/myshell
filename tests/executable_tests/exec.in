python3 tests/executable_tests/exec.py
exit
