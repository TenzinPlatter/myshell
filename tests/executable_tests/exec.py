print("Hello from python!")
print("Exiting...")
print("Bye!")
