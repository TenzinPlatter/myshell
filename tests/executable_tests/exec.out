>> Hello from python!
Exiting...
Bye!
>> 